ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3014,6 +3014,12 @@ literals:
 - lighting_color_hsl_hue_angle_setpoint:
     fixed_min: 0.0
     fixed_max: 360.0
+- lighting_color_temperature_setpoint:
+    flexible_min: 0.0
+    flexible_max: 10000.0
```

Comment:
A max of 10,000 K is equivalent to 9726 degrees Celsius, which seems quite high

Reply:
Indeed. Changed to max. 7k. Should be sufficient.